### PR TITLE
Menu and Markdown Updates

### PR DESCRIPTION
**Pull request by fjebaker**

### CODE REVIEW

**Comment by fjebaker** [comment]
@Shellywell123 can you merge main into this branch on your machine `git checkout bens-safe-space && git pull origin main`, and then see if it works alright? If not, push the merge commit to this branch and then I can make the changes

I think your changes to game state might have broken the win condition

File: src/help_menu.fnl [hunk 1]
```diff
@@ -34,11 +34,24 @@
...
 
         (love.graphics.print "Kafkaesque - Pause Menu - Help." 100 50)
 
-        (love.graphics.print "You have awoken as a beetle in a world you once knew, you now feel lost \n and vlnerable. Find food and a way to escape this domestic prison...\n\nUse 'W A S D' or arrow keys crawl to freedom." 100 200)
+        ;CHANGING FONT FOR THIS PARA
+        (var font (love.graphics.newFont "assets/fonts/NugoSansLight-9YzoK.ttf" 20))
+        (love.graphics.setFont font)      
+
+        (love.graphics.print "Help I'm a beetle?
+You have awoken as a beetle in a forgein world you once knew.
```
Comment (jwiggins) [inline src/help_menu.fnl:46]: Typo: forgein -> foreign

Comment (Shellywell123) [reply src/help_menu.fnl:46]: well noticed!

**Comment by fjebaker** [comment]
Okay, win condition fixed

**Pull request merged**